mathics/builtin/options.py

```diff
@@ -401,6 +429,49 @@ def apply_3(self, f, optvals, optname, evaluation):
             return Symbol(name)
         return val
 
+    # FIXME until we figure out what test/test_evaluation.py fails
+    apply_3 = eval_3
```

Comment:
@mmatera there is something incomplete regarding `def eval..` when there is an (error) message printed. This can be seen by removing this line and running pytest on `test/test_evaluation.py` https://github.com/Mathics3/mathics-core/actions/runs/3390463097/jobs/5634637319#step:6:4793 shows this failure.